Gemfile

```diff
@@ -1,20 +1,21 @@
 source "https://rubygems.org"
 
+gem "activemodel-serializers-xml"
```

Comment:
Where was the error being generated that caused you to add this? The class in question was removed from Rails 5 so seems odd to be popping up now.

Reply:
I was getting this error `uninitialized constant ActiveModel::Serializers::Xml`, can't exactly remember where.

Reply:
Need to track this down - try removing the gem and seeing what the backtrace is.

Reply:
Ok, running it now without that gem I don't seem to be getting the error again. From my commit history it seems it appeared when I ran `bundle update rails`. But now the error's gone I'll remove it